src/modules/block/components/blocks/blocks.css

```diff
@@ -42,3 +42,29 @@
     text-align: right;
   }
 }
+
+.header {
+  width: 100%;
+  height: calc(var(--header-height) + 45px);
+
+  & > h1 {
+    margin-bottom: 12px;
+  }
+
+  & ~ div:nth-child(2) {
+    position: sticky;
+    top: 0;
+    background-color: var(--color-body-bg);
+    z-index: 4;
+    width: fit-content;
+    margin-top: -95px;
+    margin-left: calc(100% - 119px);
+    padding: 22px 0px;
+    padding-bottom: 20px;
+  }
+
+  & ~ div:nth-child(3) > header:first-child {
```

Comment:
Same here

Reply:
the same reason as above also applies here too